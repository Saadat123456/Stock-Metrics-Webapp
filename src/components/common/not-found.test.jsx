import { render, screen } from '@testing-library/react';
import NotFound from './not-found';

test('Page Not Found', async () => {
  const { container } = render(<NotFound />);
  expect(container).toMatchSnapshot();
  expect(await screen.findByText(/OOOPS!!! PAGE NOT FOUND/i)).toBeInTheDocument();
});
