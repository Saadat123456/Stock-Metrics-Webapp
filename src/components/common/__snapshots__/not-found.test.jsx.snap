// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Not Found 1`] = `
<div>
  <div>
    OOOPS!!! PAGE NOT FOUND
  </div>
</div>
`;
